jenkins/bash_scripts/add_build_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
This isn't the right approach; you can delete this file in the next iteration of the publishing.